core/chains/evm/client/rpc_client.go

```diff
@@ -404,167 +485,9 @@ func (r *rpcClient) CallContext(ctx context.Context, result interface{}, method
 	return err
 }
 
-func (r *rpcClient) BatchCallContext(rootCtx context.Context, b []rpc.BatchElem) error {
```

Comment:
Could you move the code to minimize the diff? It's extremely difficult to review this part of the changes

Reply:
Moved